packages/fxa-payments-server/src/components/ProviderDetails/index.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { Localized } from '@fluent/react';
+import { Customer } from '../../store/types';
+import * as Provider from '../../lib/provider';
+
+import './index.scss';
+
+type ProviderDetailsProps = {
+  customer: Customer;
+};
+
+export const ProviderDetails = ({ customer }: ProviderDetailsProps) => {
```

Comment:
Maybe PaymentProviderDetails?  Partly because Provider is a thing in React also.

Reply:
yep makes sense - will update